open_prices/products/management/commands/run_update_listener.py

```diff
@@ -20,7 +20,7 @@ def process_redis_update(self, redis_update: RedisUpdate):
         if redis_update.product_type == "food":
             flavor = Flavor.off
         elif redis_update.product_type == "beauty":
-            redis_update.obf
+            flavor = Flavor.obf
```

Comment:
This means that Redis sends create & update info for all flavors, not just food ?